airflow/cli/commands/dag_command.py

```diff
@@ -294,38 +294,46 @@ def dag_next_execution(args):
 
     if dag.get_is_paused():
         print("[INFO] Please be reminded this DAG is PAUSED now.", file=sys.stderr)
+        with create_session() as session:
+            last_parsed_dag: DagModel = (
+                session.query(DagModel).filter(DagModel.dag_id == dag.dag_id)
+            ).one_or_none()
```

Comment:
Can this not be None? If not (suggested by the type hint), it should use `one` instead of `one_or_none`.

Reply:
Also `dag.get_is_paused` actually also relies on DagModel, so we should likely merge the two into one query.

Reply:
Right! My code changes use the `DagModel` only to avoid querying the model twice.